### ISSUE TITLE
Overfitting problem

### ISSUE THREAD

**Issue by NeoLiu02** [XPFly1989/FCRN#11]
Excuse me, is there any method that you use to prevent overfitting problem?
When I train this network, the train loss is good but the val loss is a lot larger.
Any help is appreciated please.
